phpseclib/Crypt/Common/SymmetricKey.php

```diff
@@ -2006,17 +2015,17 @@ private function setupMcrypt()
     protected function pad($text)
     {
         $length = strlen($text);
-
+        if ($length % $this->block_size == 0) {
+            return $text;
+        }
         if (!$this->padding) {
-            if ($length % $this->block_size == 0) {
-                return $text;
-            } else {
-                throw new \LengthException("The plaintext's length ($length) is not a multiple of the block size ({$this->block_size}). Try enabling padding.");
-            }
+            throw new \LengthException("The plaintext's length ($length) is not a multiple of the block size ({$this->block_size}). Try enabling padding.");
         }
 
         $pad = $this->block_size - ($length % $this->block_size);
-
+        if ($this->custom_padding_str != null){
```

Comment:
Nit picky but there should be a space in the middle of `){`. For otherwise good PR's I'd fix issues like this myself but this PR has other issues that need resolution so while you're in there fixing those other issues might as well fix this issue as well!